java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -2892,5 +2893,34 @@ public String getTypeName() {
         return Bundle.getMessage("TypeName_Turnout");
     }
 
+    // Tooltip support
+    protected ToolTip _tooltip;
+    protected boolean _showTooltip;
+
+    public void setShowToolTip(boolean set) {
+        _showTooltip = set;
+    }
+
+    public boolean showToolTip() {
```

Comment:
A suggestion is to rename this method to `isShowToolTip()`. It makes it more clear that it returns the value of a flag (true/false).

Reply:
That section of code was copied from PositionableLabel.  For consistency it should remain as is.

Reply:
I agree.